markdown/developers/tutorials/nf_core_configs_outside_nf_core.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Tutorial: Using nf-core configs outside nf-core'
+subtitle: Guidance on how to use nf-core's centralised institutional configs in your own workflows.
+---
+
+When you've been working with nf-core pipelines for a while, you often will get very used to many of the convenience functionality offered by the wider nf-core infrastrucutre.
+
+One such thing is the centralised nf-core/configs repository of pre-configured configuration files that allow nf-core pipelines to run optimally on institutional clusters via the `-profile` parameter, e.g. `-profile uppmax`.
```

Comment:
Can we add the link to the summary of which institutional profiles are already set? (https://github.com/nf-core/configs)
And maybe redirect also to the other tutorial if their institution isn't there